magellan-sample-advanced/src/main/java/com/wealthfront/magellan/sample/advanced/MainMenuStep.kt

```diff
@@ -0,0 +1,29 @@
+package com.wealthfront.magellan.sample.advanced
+
+import android.content.Context
+import com.wealthfront.magellan.core.Step
+import com.wealthfront.magellan.sample.advanced.SampleApplication.Companion.app
+import com.wealthfront.magellan.sample.advanced.databinding.MainMenuBinding
+import javax.inject.Inject
+
+class MainMenuStep(
+  private val goToCollection: () -> Unit,
+  private val goToDesignCereal: () -> Unit,
+  private val goToOrderTickets: () -> Unit,
+  private val goToRequestExhibit: () -> Unit
+) : Step<MainMenuBinding>(MainMenuBinding::inflate) {
+
+  @Inject lateinit var toolbarHelper: ToolbarHelper
+
+  override fun onCreate(context: Context) {
+    app(context).injector().inject(this)
```

Comment:
Not right now, but we may want to make something here nullable, so that this doesn't break in tests